Languages/IronPython/IronPython/Runtime/Operations/LongOps.cs

```diff
@@ -52,12 +52,20 @@ public static partial class BigIntegerOps {
         }
 
         [StaticExtensionMethod]
-        public static object __new__(CodeContext/*!*/ context, PythonType cls, IList<byte> s) {
+        public static object __new__(CodeContext/*!*/ context, PythonType cls, IList<byte> s)
```

Comment:
Please maintain the formatting of the file. Formatting changes make it difficult to see what the actual fix is.